lib/tpm2_policy.c

```diff
@@ -391,7 +391,8 @@ tool_rc tpm2_policy_build_policysigned(ESYS_CONTEXT *ectx,
         tpm2_session *policy_session, tpm2_loaded_object *auth_entity_obj,
         TPMT_SIGNATURE *signature, INT32 expiration, TPM2B_TIMEOUT **timeout,
         TPMT_TK_AUTH **policy_ticket, const char *policy_qualifier_data,
-        bool is_nonce_tpm, const char *raw_data_path) {
+        bool is_nonce_tpm, const char *raw_data_path,
+        const char* cp_hash_a_path) {
```

Comment:
Attach the asterisk to the identifier — const char *cp_hash_a_path.

Suggestion: The identifier looks overly snake_cased — path_cphash ? —